airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/destination/DestinationAcceptanceTest.kt

```diff
@@ -459,25 +466,17 @@ abstract class DestinationAcceptanceTest {
                 Jsons.deserialize(it, io.airbyte.protocol.models.v0.AirbyteMessage::class.java)
             }
 
-        val largeNumberRecords =
-            Collections.nCopies(400, messages)
-                .flatten() // regroup messages per stream
-                .sortedWith(
-                    Comparator.comparing { obj: io.airbyte.protocol.models.v0.AirbyteMessage ->
-                            obj.type
-                        }
-                        .thenComparing { message: io.airbyte.protocol.models.v0.AirbyteMessage ->
-                            if (
-                                message.type ==
-                                    io.airbyte.protocol.models.v0.AirbyteMessage.Type.RECORD
-                            )
-                                message.record.stream
-                            else message.toString()
-                        }
-                )
+        /* Replicate the runs of messages and state hundreds of times, but keep trace messages at the end. */
+        val lotsOfRecordAndStateBlocks =
```

Comment:
It was pushing all the state messages to the end. We explicitly don't want that anymore, because we're verifying checkpointing.

Reply:
(Specifically, I changes the tests so that they check all the returned state messages and validate their counts instead of just checking for the last one.)